lib/java/com/google/android/material/floatingactionbutton/ExtendedFloatingActionButton.java

```diff
@@ -217,6 +234,21 @@ protected void onAttachedToWindow() {
     return behavior;
   }
 
+  /**
+   * Determines how the extended button is hidden when it needs to be collapsed
+   * @param isHide If true, none of the button will be visible when collapsed. If false, the button
+   * will "shrink" and hide button text but keep the icon.
+   *
+   * This option is "false" and perform a shrink by default.
+   */
+  public void setHideBehavior(boolean isHide) {
+
+    ExtendedFloatingActionButtonBehavior<ExtendedFloatingActionButton> extendedBehavior = (ExtendedFloatingActionButtonBehavior<ExtendedFloatingActionButton>) behavior;
```

Comment:
We should probably just change the type of the behavior to be `ExtendedFloatingActionButtonBehavior`. I don't think it would be anything else.

Reply:
Okay, I will add that in...